tests/unit_tests/status/test_tracking_integration.py

```diff
@@ -40,6 +40,12 @@ def check_expression(original, path_expression, expected, msg_start):
     match_found = False
     for match in jsonpath_expr.find(original):
         match_found = True
+
+        if match.value != expected:
+            import pdb
+
+            pdb.set_trace()
```

Comment:
ah, this is not in ready for review yet :)

Reply:
;)